tests/unit/database_tests.py

```diff
@@ -34,6 +34,7 @@
 from cloudant.design_document import DesignDocument
 from cloudant.index import Index, TextIndex, SpecialIndex
 from cloudant.feed import Feed, InfiniteFeed
+from mock import mock
```

Comment:
I would say that organizationally this import should be up with the other Python libs rather than after the cloudant imports.  So move this import to sit after `import unittest` and before `import requests`